editions/tw5.com/tiddlers/widgets/WikifyWidget.tid

```diff
@@ -44,3 +44,7 @@ The available output types are:
 !! Exporting rendered html
 
 <<testcase "TestCases/WikifyWidget/RenderHTML">>
+
+!! Rendering parametrized svg for use in stylesheets
```

Comment:
There's a typo, but our house style is to generally use British English spellings.

Reply:
Sorry about that! I fixed it.